docs/administrators/authorization.rst

```diff
@@ -0,0 +1,275 @@
+
+    # Don't allow anything that is not explicitly allowed.
+    default allow := false
+
+    # Give read-only access to users with the read-only role.
+    allow if {
+        input.token.role == "read-only"
+        endpoint_data.read_only == true
+    }
+
+    # Users with the user role are allowed to call any API endpoint.
+    allow if {
+        input.token.role == "user"
+    }
+
+
+Default access policy
+^^^^^^^^^^^^^^^^^^^^^
+
+The Inmanta server comes with a default policy that is defined in ``/etc/inmanta/authorization/policy.rego``. The policy assumes that there are two types of roles: environment-scoped roles and global roles. As the same suggests, environment-scoped roles are only relevant within a specific environment. Global roles are relevant on the entire Inmanta server. The default policy defines the following roles:
+
+1. Environment-scoped roles:
+
+   * **read-only**: Users with this role have read-only access on everything in a certain environment.
+   * **noc**: Users with this role can do anything in a certain environment, except for modifying the desired state, changing environment settings and expert operations.
+   * **operator**: Users with this role can create, update and delete service instances in a certain environment next to all the actions allowed by the noc role.
+   * **environment-admin**: Users with this role can do anything in a certain environment, except for expert operations.
+   * **environment-expert-admin**: Users with this role can do anything in a certain environment.
+
+2. Global admin role: A user with this role can execute any API endpoint on the Inmanta server.
+
+The default policy makes the following assumptions about the content of the access token:
+
+* The ``sub`` claim is present and contains the username of the user the token is for.
+* Environment-scoped roles are defined in the ``urn:inmanta:roles`` claim of the access token. The value must be a dictionary that maps the ID of the environment to a list of roles the user has in that environment.
+* Global admins must have the claim ``urn:inmanta:is_admin`` in the access token with the value set to ``true``.
+
+
+Integration with database authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The default policy integrates seamlessly with database authentication.
+
+* The admin user created using the ``/opt/inmanta/bin/inmanta-initial-user-setup`` command will have the ``urn:inmanta:is_admin`` claim set to true.
+* The web-console provides support to manage users and role assignments.
+
+If the policy contains a ``roles`` variable that contains a list role names, the Inmanta server will create these roles automatically when the server starts. The default policy defines this variable for the roles it can handle. Like that there is no need to create roles by hand. The snippet below provides an example. Removing a role from this list will not remove that role when the server starts. That should be done manually.
```

Comment:
```suggestion
If the policy contains a ``roles`` variable that contains a list role names, the Inmanta server will create these roles automatically when the server starts. The default policy defines this variable for the roles it can handle. That way there is no need to create roles by hand. The snippet below provides an example. Removing a role from this list will not remove that role when the server starts. That should be done manually.
```